docs/alerts.md

```diff
@@ -51,15 +129,91 @@ Elasticsearch will attempt to relocate shards away from a node for [reaching the
+              ```
+        - If nothing above helps, then last solution is to delete old indices
+            - Check the status of all indices on elasticsearch:
+              ```
+              oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- indices
+              ```
+            - Identify the index that can be deleted.
+            - Delete the index:
+              ```
+              oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name> -X DELETE
+              ```
+    
 ## Elasticsearch Node Disk Flood Watermark Reached
 
  Elasticsearch enforces a read-only index block on every index that has one or more shards allocated on the node, and that has at least one disk exceeding the [flood stage](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/disk-allocator.html).
 
 ### Troubleshooting
 
-Check the disk space of the node.
+1. Check the disk space of the Elasticsearch node:
+   ```
+   oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_nodes/stats/fs?pretty
+   ```
+   Check the “nodes.node_name.fs” field to determine the free disk space on that node.
+
+2. If used disk percentage is above 95% then:
+    - It signifies that you have crossed flood watermark already and writing will be blocked for shards allocated on this particular node.
+    - Increase disk space on all nodes.
+    - If increasing disk space isn’t possible, then add a new data node to the cluster.
+    - If adding a new data node is problematic, then reduce the number of replicas
+        - Preferably on older indices.
+        - Get the index settings:
+          ```
+          oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_settings?pretty
+          ```
+        - Identify the number of replicas from the output of the above command.
+        - Lower the number of replicas if possible:
+          ```
+          oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_settings?pretty -X PUT -d '{"index.number_of_replicas":<number_of_replicas>}'
+          ```
+    - If nothing above helps, then last solution is to delete old indices
+        - Check the status of all indices on elasticsearch:
+          ```
+          oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- indices
+          ```
+        - Identify the index that can be deleted.
+        - Delete the index:
+          ```
+          oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name> -X DELETE
+          ```
+    - After freeing up the disk space, when used disk percentage drops below 90% then:
```

Comment:
we can also add a step to reduce the number of replicas here similar to the data trend below

Reply:
That step is already present.